src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Web/Program.cs

```diff
@@ -14,10 +14,12 @@ namespace OrchardCore.Templates.Cms.Web
 {
     public class Program
     {
-        public static Task Main(string[] args)
-            => BuildHost(args).RunAsync();
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
 
-        public static IHost BuildHost(string[] args) =>
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
                 .ConfigureLogging(logging => logging.ClearProviders())
```

Comment:
This breaks automatic startup of the website in Visual Studio Code.

Reply:
@Skrypt  not sure what you mean by automatic startup?  the Launch task works well with F5

Reply:
It should open up your default browser automatically in the end. When I comment out this line it does work back.

Reply:
updated to `launch.json` to use `serverReadyAction`